lazy_tensor_core/lazy_tensor_core/csrc/LazyShapeInference.cpp

```diff
@@ -212,6 +220,14 @@ std::vector<Shape> compute_shape_masked_fill_(at::Tensor & self, const at::Tenso
   return {Shape(self.scalar_type(), self.sizes().vec())};
 }
 
+std::vector<Shape> compute_shape_max(const at::Tensor & self) {
+  return {Shape(self.scalar_type(), {})};
```

Comment:
I think min/max shape funcs should include the check for numel>0 in ReduceAllOps.cpp

  TORCH_CHECK(self.numel() > 0,
              "max(): Expected reduction dim to be specified for input.numel() == 0. Specify the reduction dim with the 'dim' argument.");

Reply:
similar check in min?

Reply:
Duh, thanks!